src/js/components/List/List.js

```diff
@@ -225,17 +227,17 @@ const List = React.forwardRef(
               ? () => {
                   const min = onOrder ? 1 : 0;
                   setActive(Math.max(active - 1, min));
+                  setLastActive(active);
                 }
               : undefined
           }
           onDown={
             (onClickItem || onOrder) && data && data.length
               ? () => {
                   const min = onOrder ? 1 : 0;
-                  const max = onOrder ? (data.length * 2) - 2 : data.length - 1;
-                  setActive(
-                    active >= min ? Math.min(active + 1, max) : min,
-                  );
+                  const max = onOrder ? data.length * 2 - 2 : data.length - 1;
+                  setActive(active >= min ? Math.min(active + 1, max) : min);
+                  setLastActive(active);
```

Comment:
I don't think we need to `setLastActive` here.

Reply:
Also removed this from `onUp` as well and everything still works as intended!